docs/source/satkill_explanation.rst

```diff
@@ -0,0 +1,240 @@
+:orphan:
+
+============================
+Satellite killer explanation
+============================
+
+
+
+The `functions in question <https://github.com/next-exp/IC/blob/master/invisible_cities/reco/deconv_functions.py#L26>`_:
+
+
+.. highlight:: python 
+.. code-block:: python
+
```

Comment:
I'm hesitant to include the whole code here. If we update the IC code, this block will be updated while the overall explanation might still be valid. Perhaps it's better to just point to the relevant functions in the file and let the reader read them in parallel.

Reply:
Should I add a reference to the specific commit so that if it does change, it is clear that it's due to new commits?